packages/plugin-core/src/test/suite-integ/GoToSibling.test.ts

```diff
@@ -1,415 +1,363 @@
-import { DVault, NoteUtils } from "@dendronhq/common-all";
-import { note2File, vault2Path } from "@dendronhq/common-server";
-import { runJestHarnessV2 } from "@dendronhq/common-test-utils";
-import _ from "lodash";
-import path from "path";
-import * as vscode from "vscode";
+import { DVault, WorkspaceOpts } from "@dendronhq/common-all";
+import {
+  CreateNoteOptsV4,
+  NoteTestUtilsV4,
+} from "@dendronhq/common-test-utils";
+import { describe } from "mocha";
 import { GoToSiblingCommand } from "../../commands/GoToSiblingCommand";
+import { IDendronExtension } from "../../dendronExtensionInterface";
+import { ExtensionProvider } from "../../ExtensionProvider";
 import { VSCodeUtils } from "../../vsCodeUtils";
+import { WSUtilsV2 } from "../../WSUtilsV2";
 import { expect } from "../testUtilsv2";
-import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+import { describeMultiWS, describeSingleWS } from "../testUtilsV3";
 
-suite("GoToSibling", function () {
-  const direction = "next" as const;
-  const ctx = setupBeforeAfter(this, {});
+const createNotes = async ({
+  opts,
+  fnames,
+}: {
+  opts: Omit<CreateNoteOptsV4, "fname">;
+  fnames: string[];
+}) => {
+  for (const fname of fnames)
+    await NoteTestUtilsV4.createNote({ ...opts, fname });
+};
 
-  const createNotes = (wsRoot: string, vault: DVault) => {
-    return Promise.all([
-      note2File({
-        note: NoteUtils.create({ fname: "foo.journal.2020.08.29", vault }),
-        vault,
-        wsRoot,
-      }),
-      note2File({
-        note: NoteUtils.create({ fname: "foo.journal.2020.08.30", vault }),
-        vault,
-        wsRoot,
-      }),
-      note2File({
-        note: NoteUtils.create({ fname: "foo.journal.2020.08.31", vault }),
-        vault,
-        wsRoot,
-      }),
-    ]);
+const getPostSetupHookForNonJournalNotes =
+  (fnames: string[]) =>
+  async ({ wsRoot, vaults }: WorkspaceOpts) => {
+    await createNotes({
+      opts: { wsRoot, vault: vaults[0] },
+      fnames: fnames,
```

Comment:
just use `fnames` if value is same as arg. this should be caught by eslint. make sure [[Setup ESLint|dendron://dendron.docs/dev.process.code.dev-env#setup-eslint]] is installed